assets/js/data/validation/actions.ts

```diff
@@ -35,10 +35,21 @@ export const showAllValidationErrors = () => ( {
 	type: types.SHOW_ALL_VALIDATION_ERRORS,
 } );
 
+/**
+ * Clears validation errors for the given ids.
+ *
+ * @param  errors Array of error ids to clear.
+ */
+export const clearValidationErrors = ( errors: string[] ) => ( {
+	type: types.CLEAR_VALIDATION_ERRORS,
+	errors,
+} );
+
 export type ValidationAction = ReturnOrGeneratorYieldUnion<
 	| typeof setValidationErrors
 	| typeof clearAllValidationErrors
 	| typeof clearValidationError
+	| typeof clearValidationErrors
```

Comment:
Nice idea actually. We should deprecate `clearAllValidationErrors` if we go ahead with this (instead of removing it) since it's currently out in the wild

Reply:
Yeah good point. I think it'll neaten things up. Do you want to do it as part of this PR? What's the process for deprecating things?